CMakeLists.txt

```diff
@@ -4,10 +4,30 @@
 
 cmake_minimum_required(VERSION 3.16)
 
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED 1)
+set(CMAKE_CXX_STANDARD 11 CACHE STRING "")
```

Comment:
This change allows applications to build with newer c++ standards while leaving 11 as the default.